sidekick_core/pubspec.yaml

```diff
@@ -11,7 +11,12 @@ dependencies:
   args: ^2.1.0
   dartx: '>=0.7.1 <2.0.0'
   dcli: ^1.15.0
+  deep_pick: ^0.10.0
+  http: ^0.13.5
   meta: ^1.5.0
+  pub_semver: ^2.1.3
+  # transitive from dcli
+  pubspec2: ^2.4.1
```

Comment:
We should be very cautious when adding new packages to the core because it is shipped with every sidekick cli.

For sure we need `http`.
`pub_semver` + `pubspec2` are already in `dcli`, fine.

But we can easily get rid of `deep_pick`.